cmd/triggerRun/cmd/root_test.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2020 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"encoding/json"
+	"net/http/httputil"
+	"regexp"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestReadTrigger(t *testing.T) {
+	tri, err := readTrigger("../testdata/trigger.yaml")
+	if err != nil {
+		t.Fatal("", err)
+	}
+
+	bytes, err := json.Marshal(tri)
+	if err != nil {
+		t.Fatal("", err)
+	}
+	str := string(bytes)
+
+	expect := `[{"kind":"Trigger",` +
```

Comment:
Go supports raw strings using `. We should use those instead of using + to append strings.

Reply:
https://yourbasic.org/golang/multiline-string/#raw-string-literals